grid/recruitment/update-candidate/profile.supr

```diff
@@ -0,0 +1,397 @@
+"""
+Update Candidate
+Update existing candidate application in ATS (Applicant Tracking System).
+"""
+name = "recruitment/update-candidate"
+version = "1.0.0"
+
+"""
+Update Candidate
+Updates candidate application for company job. Specify candidate fields in the input to be updated.
+"""
+usecase UpdateCandidate unsafe {
+  input {
+    """
+    Candidate ID
+    Id of a candidate to be updated.
+    """
+    candidateId! string!
+
+    """
+    First Name
+    First name of a candidate.
+    """
+    firstName string!
+
+    """
+    Last Name
+    Last name of a candidate.
+    """
+    lastName string!
+
+    """
+    Email
+    Email of a candidate.
+    """
+    email string!
+
+    """
+    Phone
+    Phone number of a candidate.
+    """
+    phone string
+
+    """
+    Address
+    Address of a candidate.
+    """
+    address string	
+
+    """
+    Education
+    Array of education experiences.
+    """
+    education [Education]
+
+    """
+    Work Experience
+    Array of work experiences.
+    """
+    workExperience [WorkExperience]
+
+    """
+    CV (Resume)
+    Object containing either URL to a CV or data with CV document (BinaryData type) and fileName with the file name.
+    """
+    cv { url string } | { fileName string, data }
+
+    """
+    Links
+    Array of links to different profiles or portfolios of a candidate.
+    """
+    links [Link]
+
+    """
+    Answers
+    Array of answers to questions connected to a job that candidate applies to.
+    """
+    answers [Answer]
+
+    """
+    Stage Id
+    Unique identifier of a stage, in which candidate is in.
+    """
+    stageId string
+  }
+
+  result {
+    rateLimit RateLimit
```

Comment:
the success result is rate limit?

Reply:
It is update operation, so the result would be `void`, but as we return `rateLimit` data with each perform, we have it here as well. It might be confusing.